examples/dataservice-read/example.cpp

```diff
@@ -55,6 +114,7 @@ int main() {
   auto serviceClient = std::make_unique<olp::dataservice::read::CatalogClient>(
       olp::client::HRN(gCatalogHRN), settings);
 
+  std::string firstPartitionId;
   {  // Retrieve the catalog metadata
```

Comment:
do we need extra scope here?

Reply:
to show borderline of catalog request part and get rid of its request variables